cmonge/trainers/ot_trainer.py

```diff
@@ -13,19 +13,24 @@
     log_point_clouds,
 )
 from cmonge.metrics import fitting_loss, regularizer
+from cmonge.models.mestibator import EarlyStopMapEstimator
 from cmonge.utils import create_or_update_logfile, optim_factory
 from dotmap import DotMap
 from flax import linen as nn
 from jax.lib import xla_bridge
 from loguru import logger
 from ott.solvers.nn import models, neuraldual
 from ott.tools import map_estimator
+from orbax.checkpoint import CheckpointManagerOptions, CheckpointManager
+from orbax.checkpoint.args import StandardSave
 
 
 class AbstractTrainer:
     """Abstract class for neural OT traininig."""
 
-    def __init__(self, jobid: int, logger_path: Path) -> None:
+    def __init__(
+        self, jobid: int, logger_path: Path, mlflow_logging: bool = False
```

Comment:
Now you're passing the variable but you're not registering as attribute in the constructor